test/tests.ts

```diff
@@ -310,6 +310,20 @@ export function testNitro(
     expect(data).toMatch("<h1 >Hello JSX!</h1>");
   });
 
+  it.skipIf(ctx.preset.startsWith('cloudflare') || ctx.preset === 'vercel-edge')(
```

Comment:
Is the issue with this two diff in space? (we can normalize `headers['vary']` to remove space)

Reply:
Here I had something even different with Accept-Encoding not applying at all. I guess it depends on serve static param

Reply:
Ah right.